Address action-1228

aphillips opened the pull request:
* add recommendation to use our character naming template and official names
* add examples of U+ notation
* clean up text


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/bp-i18n-specdev/pull/86.html" title="Last updated on Dec 15, 2022, 5:21 PM UTC (f0a14c7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/86/ba2541e...aphillips:f0a14c7.html" title="Last updated on Dec 15, 2022, 5:21 PM UTC (f0a14c7)">Diff</a>

aphillips committed:
address action-1228

* add recommendation to use our character naming template and official names
* add examples of U+ notation
* clean up text

xfq commented on index.html:
```diff
@@ -2056,14 +2056,74 @@ <h3>Referring to Unicode characters</h3>
+</aside>
+
+<div class="req" id="char_ref_unicode_names">
+	<p class="advisement">Use the Unicode character name to describe specific code points. Use of the <a href="">character naming template</a> is RECOMMENDED.</p>
```
The link to "character naming template" is currently missing.

aphillips replied:
Fixed.

xfq commented on index.html:
```diff
@@ -2056,14 +2056,74 @@ <h3>Referring to Unicode characters</h3>
 
+<p>There are cases where doing this is overly pedantic and detracts from usability, but be cautious about being so informal as to impair meaning.</p>
 
+<p>Internationalization specifications use (and we recommend the use of) this template for character references:</p>
```
Perhaps we can add a link to https://www.w3.org/International/i18n-activity/guidelines/editing#codepoints ?

aphillips replied:
Added. Thanks!

aphillips committed:
Address @xfq's comments

Add link to i18n editing guidelines.

Add missing internal link to the template.

Also added a section around the character naming template and some notes on markup.

aphillips committed:
Merge branch 'w3c:gh-pages' into action-1228

aphillips committed:
Changes per telecon of 2022-12-15

* Fix typo spotted by Bert
* change example language tag to 'fr'
* Remove sentence about what language tags to use. @r12a promised to guide confused spec writers :-D

aphillips commented:
Merging per telecon of 2022-12-15.
